lib/install.js

```diff
@@ -443,10 +443,16 @@ function download (gyp, env, url) {
               || env.http_proxy
               || env.HTTP_PROXY
               || env.npm_config_proxy
+  
+  var strictSsl = true;
+  if (!gyp.opts.strictssl) {
+    strictSsl = gyp.opts.strictssl
+  }
```

Comment:
That always disables SSL verification unless you explicitly pass `--strictssl=true`, doesn't it..?